content/en/docs/concepts/security/index.md

```diff
@@ -541,13 +538,19 @@ authorization policies using `.yaml` files.
 
 ### Implicit enablement
 
-You don't need to explicitly enable Istio's authorization features. Just apply
-an authorization policy to the workloads to enforce access control.
-For workloads without authorization policies applied, Istio doesn't enforce
-access control allowing all requests.
+You don't need to explicitly enable Istio's authorization features; they are available after installation.
```

Comment:
Does some of the content belong to the security tasks? I feel the concept page is pretty long.